omnipaxos_core/src/omni_paxos.rs

```diff
@@ -81,6 +89,8 @@ impl Default for OmniPaxosConfig {
             pid: 0,
             peers: Vec::new(),
             buffer_size: BUFFER_SIZE,
+            election_tick_timeout: 100,
+            resend_message_tick_timeout: 500,
```

Comment:
Define these as constants. Not sure where BUFFER_SIZE is right now but all default values should be placed together somewhere where it makes sense